src/components/Insight/InsightEditor/InsightEditor.js

```diff
@@ -31,6 +31,10 @@ class InsightEditor extends Component {
     isEditing: false
   }
 
+  componentDidUpdate ({ tags }, { tags: stateTags }) {
+    if (tags.length > 0 && this.props.tags !== tags) this.setState({ tags })
+  }
```

Comment:
`stateTags` is not being used.
@haritonasty please, can you explain why this lifecycle method is needed?

Reply:
![image](https://user-images.githubusercontent.com/24521041/60752356-fab7b900-9fcc-11e9-83b8-00a16b03c2a5.png)
sometimes new.props and new.state are empty and only from prev props we can restore tags(